galley/pkg/runtime/processing/dispatcher.go

```diff
@@ -23,13 +23,21 @@ var scope = log.RegisterScope("processing", "Galley data processing", 0)
 
 // Dispatcher is a Handler that can dispatch events to other Handlers, based on Collection.
 type Dispatcher struct {
-	handlers map[resource.Collection][]Handler
+	fullSyncHandlers []Handler
```

Comment:
What is the difference between a handler and a fullSyncHandler? Is this for incremental MCP?

Reply:
full sync is a function of the sources in Galley. In this case, it indicates that the entire configuration has been retrieved from k8s for nodes, pods, endpoints, and services.